specs/altair/beacon-chain.md

```diff
@@ -628,11 +630,14 @@ def process_justification_and_finalization(state: BeaconState) -> None:
 ```python
 def process_inactivity_updates(state: BeaconState) -> None:
     for index in get_eligible_validator_indices(state):
+        # Increase inactivity score of inactive validators
         if index in get_unslashed_participating_indices(state, TIMELY_TARGET_FLAG_INDEX, get_previous_epoch(state)):
-            if state.inactivity_scores[index] > 0:
-                state.inactivity_scores[index] -= 1
-        elif is_in_inactivity_leak(state):
+            state.inactivity_scores[index] -= min(1, state.inactivity_scores[index])
+        else:
             state.inactivity_scores[index] += INACTIVITY_SCORE_BIAS
```

Comment:
Why do we increase the inactivity score even when we're not in an inactivity leak?
Or is this one of these weird ones where it's supposed to cancel out with the line below and what we want is that it decreases by `INACTIVITY_SCORE_RECOVERY_RATE - INACTIVITY_SCORE_BIAS` for offline validators? Is that the right constant though?